doc/release-notes.md

```diff
@@ -15,3 +15,31 @@ Memory Usage Improvement
 
 The memory usage of zcashd has been reduced by not keeping Equihash solutions for all
 block headers in memory.
+
+RPC changes
+-----------
+
+The following RPC methods that query wallet state now support an optional `asOfHeight`
+parameter, to execute the query as if it were run when the blockchain was at the height
+specified by this argument:
+
+* `getbalance`
+* `getreceivedbyaddress`
+* `gettransaction` (*)
+* `getwalletinfo`
+* `listaddressgroupings`
+* `listreceivedbyaddress` (*)
+* `listsinceblock` (*)
+* `listtransactions`
+* `listunspent` (*)
+* `z_getbalanceforaccount`
+* `z_getbalanceforviewingkey`
+* `z_getmigrationstatus`
+* `z_getnotescount` (also adds `minconf` parameter)
```

Comment:
```suggestion
* `z_getnotescount`
```
`minconf` was already supported, and was listed in the long-form parameter list docs, just not in the usage summary line. See https://github.com/zcash/zcash/commit/659077aff27a70c9a4fe615060b2e4c7893398b0#diff-63c7710d72381cffe692175dd70e47f1891bf75c3e09249ed826f9df3917d02dR6396-R6399

Reply:
*sigh*, the link doesn’t work, because rpcwallet.cpp isn’t loaded by default, but the relevant bit of code in that commit starts at rpcwallet.cpp:6396 (new line numbers).